### DISCUSSION TITLE
Option to save/write screen image to disk

### DISCUSSION THREAD

**Discussion by drum445** [hajimehoshi/ebiten#2668] [Q&A] [answered]
It would be useful to have a way to save the screen after Draw() has been ran to a file on disk. Similar to raylib's screenshot functionality.

If there is already a way to achieve this, please let me know

Thanks

**Comment by hajimehoshi** [comment, marked as answer]
1. Use [`EBITENGINE_SCREENSHOT_KEY` environment variable](https://pkg.go.dev/github.com/hajimehoshi/ebiten/v2#hdr-Environment_variables).
2. You can save an `*ebiten.Image` by passing this to [`png.Encode`](https://pkg.go.dev/image/png#Encode) or something, as this implements [`image.Image`](https://pkg.go.dev/image#Image).

**Reply by drum445** [reply]
That's great, thanks @hajimehoshi 👍
